### ISSUE TITLE
[Bug]: Failed to load Reminder Plugin

### ISSUE THREAD

**Issue by kennygokh** [uphy/obsidian-reminder#189]
### Describe the bug

A message displayed when open obsidian app on macOS. Obsidian version 1.6.3.

### Expected Behavior

Reminder plugin should load as per normal. It was working fine until recent upgrade of Obsidian to v1.6.3.

### Steps to reproduce

Open obsidian app, error message remain - failed to load Obsidian Reminder. I have reinstall obsidian reminder plugin but error remain the same.

### Operating system

macOS

**Comment by kennygokh** [comment]
@uphy Do you have a chance to look into this issue? I rely a lot on reminders to help to remind me on tasks on time. Hope that you can help to resolve it 🙏

**Comment by claremacrae** [comment]
Some more info might help the author, like what error message is shown on the console?

And does the problem occur when. only this plugin is enabled?

**Comment by kennygokh** [comment]
> Some more info might help the author, like what error message is shown on the console?
> 
> And does the problem occur when. only this plugin is enabled?

Thanks @claremacrae I realised that my personal vault is working fine. I did comparison on the .json files and discovered that task format setting is not enabled. This is probably because when the issue happened, I did reinstallation and missed checking the configuration settings. After the settings checked and enabled. It seems working fine now. Thanks anyway 🙏
